frontend/src/audios.html

```diff
@@ -40,7 +54,12 @@
     </div>
     <div class="my-3 flex items-center">
       <!-- Filter button -->
-      <button data-filter-toggle class="mr-4 inline-flex w-24 items-center space-x-1 rounded-md border-gray-300 bg-neutral-600 px-4 py-2 font-semibold text-white shadow-sm md:hidden">
+      <button
+        type="button"
+        @click="open = true"
+        x-show="!open"
```

Comment:
I think we can keep it visible even when the panel is open. Anyhow the button stays in the background under the blur.

Reply:
Not fixed.

Reply:
`x-show ` removed from filter button